File: transaction-bff/src/main/java/br/com/felipe/gadelha/coffeandit/transactionbff/api/v1/dto/response/AccountRs.java
package br.com.felipe.gadelha.coffeandit.transactionbff.api.v1.dto.response;

import br.com.felipe.gadelha.coffeandit.transactionbff.domain.entity.Account;

public class AccountRs {

    private final Long id;
    private final Long agency;
    public AccountRs(Account account) {
        this.id = account.getId();
        this.agency = account.getAgency();
    }
    public Long getId() { return id; }
    public Long getAgency() { return agency; }
}

File: transaction-ms/src/main/kotlin/br/com/felipe/gadelha/coffeandit/transactionms/api/v1/dto/response/AccountRs.kt
package br.com.felipe.gadelha.coffeandit.transactionms.api.v1.dto.response

import br.com.felipe.gadelha.coffeandit.transactionms.domain.entity.Account

class AccountRs private constructor (
    val id: Long,
    val agency: String
) {
    constructor(account: Account):
        this(
            id = account.id!!,
            agency = account.agency
    )
}